README.md

```diff
@@ -24,13 +24,24 @@ We should only use static HTML and CSS for this project. Preprocessors (haml, er
 
 This group fansite must have the following HTML files:
 
-- `index.html` must include your names or team name somewhere.
+- `index.html` must include your names or team name somewhere. This file is in the project root.
 
-- `facts.html` should include a series of fun facts or more detailed information about your topic.
+- `facts.html` should include a series of fun facts or more detailed information about your topic. This file is in the `pages` folder.
 
-- `gallery.html` must include at least 3 images with descriptions and links to where the images came from if applicable. 
+- `gallery.html` must include at least 3 images with descriptions and links to where the images came from if applicable. This file is also in the `pages` folder.
 
-Each group member should take responsibility for creating one HTML file. 
+Each group member should take responsibility for completing one HTML file.
+
+##### For Groups With More Than 3 Members
+
+Add an additional page of your choice for each additional group member!
```

Comment:
It may not be necessary, but I think it could be useful for expectations to be very clear that the extra pages are a requirement for larger teams.

```suggestion
In addition to the 3 pages listed above, teams of 4 or more are required to add an additional page of your choice for each additional group member.
```